### PR TITLE
Added Security chart to enable security as an integration

### PR DESCRIPTION
**Pull request by dheeth**
- Added image-scanner
- Added clair as dependency

### CODE REVIEW

File: charts/security/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: security
```
Comment (nishant-d) [inline charts/security/Chart.yaml:2]: give some specific name like `devtron-security-integration`

File: charts/security/templates/image-scanner.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      containers:
+        - name: image-scanner
+          image: {{ .image }}
+          imagePullPolicy: {{ .imagePullPolicy }}
+          ports:
+            - name: app
+              containerPort: 8080
+              protocol: TCP
+          env:
+            - name: CONFIG_HASH
```
Comment (nishant-d) [inline charts/security/templates/image-scanner.yaml:75]: remove `CONFIG_HASH` `SECRET_HASH` from env

**Review by nishant-d — DISMISSED** [review]

File: charts/security/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+description: A Helm chart for devtron to enable security integration
+type: application
+maintaintainers:
+  - name: Pawan Kumar
+    email: pawan.mehta@devtron.ai
+sources:
+  - https://github.com/devtron-labs/charts
+version: 0.1.0
```
Comment (prakarsh-dt) [inline charts/security/Chart.yaml:10]: add keywords too

File: charts/security/templates/image-scanner.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  name: image-scanner
+  labels:
+    app: image-scanner
+    release: devtron
+spec:
+  selector:
+    matchLabels:
+      app: image-scanner
+      release: devtron
+  replicas: 1
```
Comment (prakarsh-dt) [inline charts/security/templates/image-scanner.yaml:56]: from values

File: charts/security/templates/image-scanner.yaml (hunk 2 above)
Comment (dheeth) [reply charts/security/templates/image-scanner.yaml]: Done

File: charts/security/Chart.yaml (hunk 3 above)
Comment (dheeth) [reply charts/security/Chart.yaml:18]: Added

File: charts/security/templates/image-scanner.yaml (hunk 4 above)
Comment (dheeth) [reply charts/security/templates/image-scanner.yaml]: exposed in values

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**